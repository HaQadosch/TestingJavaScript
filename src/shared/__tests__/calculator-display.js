import React from 'react'
import {render} from 'calculator-test-utils'
import CalculatorDisplay from '../calculator-display'

test('mounts', () => {
  const {container} = render(<CalculatorDisplay value="0" />)
  expect(container.firstChild).toMatchSnapshot()
})
