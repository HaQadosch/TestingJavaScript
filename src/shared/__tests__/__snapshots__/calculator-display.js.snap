// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should mount  1`] = `
<div>
  <div
    class="css-12149uf"
  >
    <div
      class="autoScalingText"
      style="transform: scale(1,1);"
    >
      0
    </div>
  </div>
</div>
`;
